packages/compass-connections/src/modules/telemetry.spec.ts

```diff
@@ -286,7 +293,7 @@ describe('connection tracking', function () {
     const trackEvent = once(process, 'compass:track');
     const connectionInfo = {
       connectionOptions: {
-        connectionString: 'mongodb://notlocalhost',
```

Comment:
This domain didn't resolve to anything, so I don't think it was a very good test. The code looks up  the IP in some sets to figure out if it is contained in there and obviously nothing that resolves would be in there. But neither of the Atlas tests higher up resolved either.